ui/src/presets/samples/mage_arcane_phase2.yml

```diff
@@ -112,14 +112,28 @@ gear:
 rotation:
   autoAttack: false
   precombat:
-    - name: Flask of Blinding Light
+    - name: Elixir of Draenic Wisdom
+    - name: Adept's Elixir
     - name: Crunchy Serpent
     - name: Arcane Intellect
-    - name: Molten Armor
+    - name: Mage Armor
```

Comment:
Would it be useful to conditionally cast Mage/Molten Armor based on fight duration?

Reply:
Not really. You basically use Mage Armor all the time.